### PR TITLE
Make get parameters work

### PR DESCRIPTION
**Pull request by andeersg**
I saw your todo "Fix ability to link to users that ajax fix above broke (e.g. http://localhost:5000/?username=jenkoian)" and I think this fixes it.

### CODE REVIEW

**Comment by jenkoian** [comment]
Awesome, thanks! One thing I noticed, it doesn't retain the username in the search bar. Also, would be *really* neat if when submitting via ajax it updated the URL also. :)

**Comment by andeersg** [comment]
I updated the pull request with those features :)

**Comment by jenkoian** [comment]
Awesome work thanks!

File: views/index.hbs [hunk 1]
```diff
@@ -33,7 +33,7 @@
             </div>
         </form>
 
-        <div id="results"></div>
+        <div id="results">{{> prs}}</div>
```
Comment (jenkoian) [inline views/index.hbs]: Sorry, been trying to look into up but can't find it...mind sharing with me what the `>` does here? I know it's important but just wondering :)

Comment (jenkoian) [inline views/index.hbs]: Ah just found this http://handlebarsjs.com/partials.html :+1:

Comment (andeersg) [inline views/index.hbs]: Yeah, partials in Handlebars are pretty nice :smile:

**Pull request merged**